custom_components/tuya_local/devices/momcozy_white_noise_machine.yaml

```diff
@@ -0,0 +1,278 @@
+name: White noise machine
+products:
+  - id: woqiobw0jfafgmar
+    name: Mom Cozy White Noise Machine, v1 - no childlock, timer 30,60,90
```

Comment:
One day this will be used for the default device name for matching product ids, so it shouldn't be too long.  I suggest "Mom Cozy white noise machine" is enough - it doesn't need to be unique, as only one of the two configs will match the device.